### PR TITLE
This is my first branch checkin

### PR DESCRIPTION
**Pull request by jasonj154**
The pull request adds the initial file for @jasonj154 in #28.

@githubstudent  I would :+1: if you could review :clock1: correctness.

### CODE REVIEW

**Comment by jasonj154** [comment]
It looks good, reviewed.

File: docs/jasonj154.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+**name:** jxj00
+
```
Comment (jasonj154) [inline docs/jasonj154.md:2]: Add more details

**Comment by sathalye** [comment]
Looks good!

File: docs/jasonj154.md (hunk 1 above)
Comment (daphnejwang) [inline docs/jasonj154.md:2]: looks good

**Pull request merged**